iothub/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt5/Mqtt5IotHubConnection.java

```diff
@@ -401,18 +431,7 @@ public boolean sendMessageResult(IotHubTransportMessage message, IotHubMessageRe
         }
 
         log.trace("Sending MQTT ACK for a received message ({})", message);
-        if (message.getMessageType() == DEVICE_METHODS)
-        {
-            // TODO
-        }
-        else if (message.getMessageType() == DEVICE_TWIN)
-        {
-            // TODO
-        }
-        else
-        {
-            // TODO
-        }
+        this.mqtt5Client.sendMessageAcknowledgement(messageId);
```

Comment:
This has been consolidated into a single method `sendMessageAcknowledgement()`, so we don't have to split into different cases.